src/v2/Apps/Auction/auctionRoutes.tsx

```diff
@@ -48,6 +49,16 @@ const AuctionFAQRoute = loadable(
 )
 
 export const auctionRoutes: AppRouteConfig[] = [
+  {
+    // Redirect from the old route to the new one
+    path: "/sale/:slug?",
+    children: [
+      new Redirect({
+        from: "/",
+        to: "/auction/:slug?",
+      }) as any,
+    ],
+  },
```

Comment:
Can we move this new code to the bottom of the route array since it will rarely be touched

Reply:
Done!